### ISSUE TITLE
Tear down containers after each test

### ISSUE THREAD

**Issue by macroking** [elgalu/docker-selenium#133]
I'm very new to docker and trying to sclae the test infrastructure using docker. 

Problem I'm facing is, Our jenkins build system gets 3-10 request each hour. We decided to run our spec files in parallel using selenium grid and docker containers. 

I created a python script to identify the number of the spec files and browser type from the current workspace and create those many containers. After that we install node modules and trigger the protractor tests.

During the execution if any new build comes up, the python script automatically determines the available number of containers not in use and create more containers if needed. Even If I spin up containers with "-rm" flag in python, I don't think after the browser session close the containers would be removed.

The challenge I'm facing here is, there are time when only few automation build request comes to jenkins. So, I'm looking for an efficient way to clean up the containers as soon the current browser session in the container ends. So that the grid will not send another test to the previous used container. This helps in ensuring a clean environment every single time.

**Comment by elgalu** [comment]
@diemol and I created a project exactly for your use case:
https://github.com/zalando-incubator/zalenium

**Comment by elgalu** [comment]
Closing as this is address by Zalenium and out of scope of this project from now on.
